src/app/components/lists/list-overview-item/list-overview-item.component.html

```diff
@@ -1,25 +1,27 @@
-<div class="row shadow mr-1 ml-1 mt-3 mt-sm-0 animated" [ngClass]="{'fadeIn':canShowToolTip}" id="main">
+<div class="modal-header cardTopRow p-0">
+    <button class="close" (click)="deleteModal.open()">
```

Comment:
Class close will give it a lot of padding which isnt right. Inspect the button and you can see the click area is way outside the actual image